server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -280,7 +280,10 @@ public CompactionStats call() throws IOException, CompactionCanceledException {
           }
         }
       } catch (IOException | RuntimeException e) {
-        log.warn("{}", e.getMessage(), e);
+        log.warn("{}", e.getMessage());
+        if (log.isDebugEnabled()) {
```

Comment:
You don't need these `if (log.isDebugEnabled())` conditionals either. You only need that when you're avoiding some expensive operation while constructing the log message. The only thing you'd avoid here is `e.getMessage()`, which I don't think is necessary.

Reply:
Good point, mostly a force of habit when doing debugging statements as sometimes it's expensive when outputting a lot of information. I'll push an update.